docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -477,5 +477,65 @@ pattern:
         json:
           name: -o json
           description: display the content of a pattern in json format
-          usage: |
-              mesheryctl pattern view [pattern-name|pattern-id] -o json
\ No newline at end of file
+          usage:
+              mesheryctl pattern view [pattern-name|pattern-id] -o json
+
+app:
+  name: app
+  description: Apps management of service meshes
+  usage:
+    mesheryctl app 
+  flags:
+    file:
+      name: --file, -f
+      description: (required) path to pattern file
+      usage:
+        mesheryctl app --file [path to pattern file]
+  subcommands:
+    list:
+      name: list
+      description: displays a list of available applications
+      usage:
+          mesheryctl app list
+    offboard:
+      name: offboard
+      description: offboard application file
+      usage:
+          mesheryctl app offboard [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: Offboard application by providing file path
+          usage:
+              mesheryctl app offboard -f [filepath] 
+    onboard:
+      name: onboard
+      description: onboard application file
+      usage: 
+          mesheryctl app onboard [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: onboard application with application file
+          usage:
+              mesheryctl app onboard --file [filepath]
+    view:
+      name: view
+      description: displays the contents of a specific pattern file
+      usage: | 
+          mesheryctl app view BookInfo (application-name|application-id)
+          OR
+          mesheryctl app view BookInfo
+      flags:
+        view-all:
+          name: --all, -a
+          description: show all application file content
+          usage:
+              mesheryctl app view --all
+        view-json:
+          name: -o json
+          description: display the content of a application in json format
+          usage: | 
+              mesheryctl app view BookInfo[application-name|application-id] -o json 
+              OR 
+              mesheryctl app view BookInfo -o json 
```

Comment:
Don't we have a separate examples section? Usage would be the skeleton of the command and not actual examples right?

Reply:
Examples section isn't made yet (for all cmds)

Reply:
There is an open issue made but no one has taken it yet

Reply:
@navendu-pottekkat FYI, #3978 

IDK who will take this issue but it'd be a good one to address it if anyone's willing to work on Jekyll

Reply:
It could be either one of them; `mesheryctl app view BookInfo -o json` looks better.